### PR TITLE
Improve formatting of changelog.txt

### PR DESCRIPTION
**Pull request by dalinaum**

### CODE REVIEW

**Comment by dalinaum** [comment]
Although 0.86.1 was already published yesterday, I fixed it too. Please review this PR @realm/java

File: changelog.txt [hunk 1]
```diff
@@ -2,20 +2,20 @@
...
+ * Fixed a bug where RealmQuery.findFirst() returned a wrong result if the RealmQuery had been created from a RealmResults.where() (#1905).
+ * Fixed a bug causing DynamicRealmObject.getObject()/setObject() to use the wrong class (#1912).
  * Fixed a bug which could cause a crash when closing Realm instances in change listeners (#1900).
- * Fix a crash occurring during update of multiple async queries (#1895).
+ * Fixed a crash occurring during update of multiple async queries (#1895).
  * Fixed listeners not triggered for RealmObject & RealmResults created using copy or create methods (#1884).
- * Fix (#1894) RealmChangeListener never called inside RealmResults.
- * Fix (#1886), crash when calling clear on a LinkView.
+ * Fixed RealmChangeListener never called inside RealmResults (#1894).
+ * Fixed crash when calling clear on a LinkView (#1886).
```
Comment (cmelchior) [inline changelog.txt]: LinkView is the internal API. This should be RealmList

**Comment by cmelchior** [comment]
Apart from small comment :+1:

**Comment by kneth** [comment]
:+1:

**Pull request merged**